_data/projects/basic/awkward-array.yml

```diff
@@ -2,8 +2,8 @@ name: awkward-array
 projlogo: /assets/images/projlogo/logo_awkward_array.png
 description: Manipulate JSON-like data with NumPy-like idioms.
 url: https://github.com/scikit-hep/awkward-1.0
-readme: https://github.com/scikit-hep/awkward-1.0/blob/main/README.md
 docs: https://awkward-array.org
+docs-title: Website
```

Comment:
You might as well use "awkward-array.org" as the title?

Reply:
```suggestion
docs-title: awkward-array.org
```

Reply:
I don’t know, I’m happy to let @jpivarski decide.

Reply:
Sure!